modules/util/portal-tools-db-partition-virtual-instance-migration-common/src/main/java/com/liferay/portal/tools/db/partition/virtual/instance/migration/common/InstanceData.java

```diff
@@ -0,0 +1,75 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.portal.tools.db.partition.virtual.instance.migration.common;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @author Luis Ortiz
+ */
+public class InstanceData {
```

Comment:
Don't use Data, just the name as in Company or Release. It can be LiferayInstance or just Instance for example